cspell.json

```diff
@@ -122,6 +122,7 @@
         "gitattributes",
         "githook",
         "gitignore",
+		"gitkeep",
```

Comment:
tabs instead of spaces?

Reply:
fixed https://github.com/prettier/prettier/pull/7320/commits/e8dbd2369d616bdf75a56d47803c7867cd0621a8